test/cmocka/src/audio/mux/demux_copy.c

```diff
@@ -88,31 +88,52 @@ static int setup_group(void **state)
 	return 0;
 }
 
+static size_t get_stream_map_size(struct test_data *td)
+{
+	size_t size = 0;
+	int i, j;
+
+	for (i = 0; i < MUX_MAX_STREAMS; ++i) {
```

Comment:
please place a comment explaining what we are calculating.

Reply:
Done